packages/salesforcedx-sobjects-faux-generator/src/generator/fauxClassGenerator.ts

```diff
@@ -17,8 +19,19 @@ import {
   SObjectCategory,
   SObjectDescribe
 } from '../describe';
+import { nls } from '../messages';
+
+export interface CancellationToken {
+  isCancellationRequested: boolean;
+}
 
+export const SUCCESS_CODE = '0';
+export const FAILURE_CODE = '1';
+const SFDX_PROJECT_FILE = 'sfdx-project.json';
```

Comment:
Is there no where else where we define this?  This seems like a useful constant that can go into some shared library.

Reply:
Yes, I moved it to a constants.ts in utils-vscode.  Missed updating this one - thanks